pkg/core/interop_neo.go

```diff
@@ -119,22 +119,22 @@ func contractUpdate(ic *interop.Context) error {
 	if contract == nil {
 		return errors.New("contract doesn't exist")
 	}
-	script := ic.VM.Estack().Pop().Bytes()
-	if len(script) > MaxContractScriptSize {
-		return errors.New("the script is too big")
+	script := ic.VM.Estack().Pop().BytesOrNil()
+	if l := len(script); script != nil && (l == 0 || l > MaxContractScriptSize) {
+		return fmt.Errorf("invalid script size: %d (max %d)", l, MaxContractScriptSize)
```

Comment:
We're differing here with C# node, it will _always_ pop two of its arguments from the stack (unless they're type-invalid like maps or interop interfaces here) and then analyze them, but we'll leave the second parameter on the stack this way.

Reply:
Fixed